protos/python/vehicle_metric_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: vehicle_metric.proto
-# Protobuf Python Version: 4.25.0
+# Protobuf Python Version: 4.25.1
```

Comment:
you would need to update https://github.com/teslamotors/fleet-telemetry/blob/main/.github/workflows/build.yml#L22 and also readme

Reply:
updated